lib/src/solar.dart

```diff
@@ -0,0 +1,71 @@
+// ignore_for_file: unnecessary_overrides
+
+import 'lunar.dart';
+
+/// An instant in time, similar to [DateTime] object, having functions
+/// that helps interacting with [Lunar] much more easier.
+class Solar implements Comparable<Solar> {
+  late final int _year;
+  int get year => _year;
+
+  late final int _month;
+  int get month => _month;
+
+  late final int _day;
+  int get day => _day;
+
+  late final int _hour;
+  int get hour => _hour;
+
+  late final int _minute;
+  int get minute => _minute;
+
+  late final int _second;
+  int get second => _second;
+
```

Comment:
I think you can define timezone in here
```dart
int get timezone = ...;
```

Reply:
The reason for setting the default timezone to GMT+7 is that this package is specifically tailored for the Vietnamese Lunar Calendar. According to the research conducted by PhD. Ho Ngoc Duc, the lunar system's calculations can vary based on the timezone (which relates to the distance from the Earth to the Moon and Sun). Therefore, using other timezones might cause the formulas to malfunction.

Thank you a lot for your contribution :D